okhttp-tests/src/test/java/com/squareup/okhttp/ConnectionSpecTest.java

```diff
@@ -166,6 +166,16 @@ public void tls_explicitCiphers() throws Exception {
     assertEquals(expectedCipherSet, expectedCipherSet);
   }
 
+  @Test
+  public void tls_customCipersAndVersions() throws Exception {
+    // Supporting arbitrary input strings allows users to enable suites and versions that are not
+    // yet known to the library, but are supported by the platform.
+    ConnectionSpec tlsSpec = new ConnectionSpec.Builder()
```

Comment:
Instead, please do `new ConnectionSpec.Builder(ConnectionSpec.MODERN_TLS)`. That way we can set TBA-future-defaults to either modern (maximum security) or compatible (maximum compatibility).